CLAUDE.md

```
@@ -207,4 +207,244 @@ GitHub Actions in `.github/workflows/ci-cd.yml`:
+- Registered in DI manually: `services.AddScoped<IUserRepository, UserRepository>()` (the caching variant is registered separately when needed)
+## Comments
+- XML doc comments (`///`) on public controller actions, response types, and shared library public APIs
+- Inline comments for non-obvious business logic — race conditions, security considerations, regression notes
+- `// ReSharper disable once` suppressions where EF Core requires private constructors
+## Module Registration
+<!-- GSD:conventions-end -->
+
+<!-- GSD:architecture-start source:ARCHITECTURE.md -->
+## Architecture
+
+## Pattern
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)